pkg/chart/chart_test.go

```diff
@@ -420,15 +424,13 @@ func (f *fakeHTTPClient) Do(h *http.Request) (*http.Response, error) {
 			// Return fake chart index (not customizable per repo)
 			body, err := json.Marshal(*f.index)
 			if err != nil {
-				fmt.Printf("Error! %v", err)
+				return nil, err
```

Comment:
Doesn't seem right to be printing to stdout and continuing on with a 200 response in this case, so verified that tests still pass if I return an error here.